standards/02-Software/02-Docs/03-Usage.md

```diff
@@ -0,0 +1,154 @@
+# Usage (WIP)
```

Comment:
This being work in progress but ready to merge means it's "good enough", and later updates are always a fair expectation. They either needn't be told it's always "work in progress", or this is not ready to be merged. We have to choose one here.
```suggestion
# Usage
```